### ISSUE TITLE
Console behaviour - end of line.

### ISSUE THREAD

**Issue by TheTanc** [questdb/questdb#85]
Whenever typing a command, users end it with `;`
As their cursor `|` is after the `;` they cannot directly execute the command and end up trying to execute the next command which is often empty. This leads to confusion, especially if the line in question is the last line of commands.

Proposals 
- when typing `;` move cursor by 1 to the left
- check if any command exists on the same line as cursor. If there is a command on the same line, execute it. If not, check the line just above.

Examples --> `|`  represents the cursor

```sql
select x from y|;
```
will execute fine because the cursor is within the command. The problem being that the cursor will be outside the command when typing normally. So users have 

```sql
select x from y;|
```
Will execute empty command (see #84 )

```sql
select x from y;
|
```
Will also execute empty.

**Comment by bluestreak01** [comment]
i will play around with web console to make execution more intuitive. #84 is fixed by the way

**Comment by leggetter** [comment]
Following the getting started guide I also came across this problem.

I wanted to get started with QuestDB so followed the guide.

When I reached the My First Database guide I copied the first command to the web console and nothing happened.

I ended up debugging the JS in the browser to try to work out why my command wasn't executing. I probably hit "Run" 10 or 20 times, tried to debug and then looked for the GitHub link on the website before finding this.

IMHO this will likely catch out quite a few people who copy & paste from the My First Database guide.

Two suggestions:

1. Fix this as a priority as it could be hurting adoption for people trying to get started
2. Provide feedback in the UI related to attempted query execution
